test/burials/config/plotAllowance.unit.spec.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm, getFormDOM } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/burials/config/form.js';
+
+describe('Burials plot allowance', () => {
+  const { schema, uiSchema, depends } = formConfig.chapters.benefitsSelection.pages.plotAllowance;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input, select, textarea').length).to.equal(7);
+  });
+
+  it('should show errors when required fields are empty', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+    submitForm(form);
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error')).length).to.equal(4);
+    expect(onSubmit.called).not.to.be.true;
+  });
+
+  it('should show actual govt contribution text field', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    formDOM.fillData('#root_govtContributionsYes', 'Y');
+
+    expect(formDOM.querySelectorAll('input, select, textarea').length).to.equal(8);
+    expect(formDOM.querySelector('#root_amountGovtContribution')).not.to.be.null;
+  });
+
+  it('should submit when all required fields are filled in', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    formDOM.fillData('#root_placeOfRemains', 'Cemetary');
```

Comment:
I asked a question in the issue about whether any of the options here are mutually exclusive.